restapi/src/main/java/io/stargate/web/docsapi/rx/RxUtils.java

```diff
@@ -15,7 +15,7 @@
  *
  */
 
-package io.stargate.web.rx;
+package io.stargate.web.docsapi.rx;
```

Comment:
Moved to new package since it was only used by Doc API (to help eventual split)

Reply:
Yes, but I hope this kind of utilities will be needed in all new stuff, as hopefully we are going reactive by default?

Reply:
If so can definitely move into shareable component -- there are some other similar things.
So what would be a good place, something like `stargate-service-common`? Or would any of existing modules work?

On very short term I just want to avoid things shared by only 2 things (REST and Doc APIs), so this move is just to note that as-is this is only used by Doc API.

Reply:
@dougwettlaufer pointed me to: https://github.com/stargate/stargate/issues/239 -- so if that existed, would probably belong there?

Reply:
Will consider as part of more general "how to reuse code".